kclvm/config/src/settings.rs

```diff
@@ -56,6 +56,7 @@ pub struct Config {
     pub disable_none: Option<bool>,
     pub verbose: Option<u32>,
     pub debug: Option<bool>,
+    pub package_maps: Option<HashMap<String, String>>,
```

Comment:
More documents to explain this field `package_maps`. Why, what and how.

Reply:
fixed.